templates/endpoints/models/meta-llama--Llama-2-7b-chat-hf_a10.yaml

```diff
@@ -43,5 +43,5 @@ scaling_config:
   placement_strategy: "STRICT_PACK"
   resources_per_worker:
     accelerator_type_a10: 0.01
-multiplex_config:
+multiplex_config:  # <-- This is required to be used as a multiplexed model
```

Comment:
It's unclear to an unfamiliar user what multiplexed is .. Something like - This is required to be able to serve multiple LoRA adapters in the same replica.